### DISCUSSION TITLE
Deployment on Huggingface

### DISCUSSION THREAD

**Discussion by zeelsheladiya** [ravi7522/Football-Prediction#22] [Ideas]
It is very easy to deploy model on hugging face rather than making a website and all

**Comment by zeelsheladiya** [comment]
So I need the opinions of you guys.

**Reply by sudipbhattarai820** [reply]
I think that would be good idea!!! lets discuss at the class

**Comment by ravi7522** [comment]
I think its a good idea to deploy it on hugging face.

**Comment by zeelsheladiya** [comment]
I created repo on hugging face space!!!! now I am waiting for model to deploy

**Comment by zeelsheladiya** [comment]
@ravi7522 @sudipbhattarai820 @babuaditya97 I need input parameters for gui in the deployment. kindly make a decision ASAP

**Reply by babuaditya97** [reply]
Hi zeel. thats awesome! by parameters you mean which features to consider?

**Comment by ravi7522** [comment]
created the model in my deployment_Ravi_DNN branch. Get it from there and try to deploy on the hugging face if you face any issue then let me know. Once its deployed on the hugging face i will create the merge request for it.

**Comment by zeelsheladiya** [comment]
First version of deployment has been deployed 🙏🏽

**Comment by zeelsheladiya** [comment]
@ravi7522 @sudipbhattarai820 @babuaditya97 I want .h5 file got saved model instead of pickle

**Comment by ravi7522** [comment]
its in testing_model_ravi branch
